testJobResultsNoSuchJob -> /req/core/job-results-exception-no-such-job: wrong API Address

Run the test (using the docker).
The test Job Results No Such Job fails.
Open the details
We obtain
Reason of Failure: ava.lang.AssertionError: Unexpected character ('<' (code 60)): expected a valid value (JSON String, Number, Array, Object or token 'null', 'true' or 'false') at [Source: (String)"<!doctype html><html lang="en"><head><title>HTTP Status 404 – Not Found</title><style type="text/css">body {font-family:Tahoma,Arial,sans-serif;} h1, h2, h3, b {color:white;background-color:#525D76;} h1 {font-size:22px;} h2 {font-size:16px;} h3 {font-size:14px;} p {font-size:12px;} a {color:black;} .line {height:1px;background-color:#525D76;border:none;}</style></head><body><h1>HTTP Status 404 – Not Found</h1><hr class="line" /><p><b>Type</b> Status Report</p><p><b>Message</b> Not Found</p><p><b"[truncated 199 chars]; line: 1, column: 2]
_____________________


The url called is:

URL:	?GET https://[OURSERVER_ENTRYPOINT]/processes/invalid-job-97ac1580-ea95-4668-8dc5-b2a4c573b435/results HTTP/1.1

But from the API doc
https://docs.ogc.org/is/18-062r2/18-062r2.html#toc34

I assume the correct address should be 
URL:	?GET https://[OURSERVER_ENTRYPOINT]/jobs/invalid-job-97ac1580-ea95-4668-8dc5-b2a4c573b435/results 

So I think that processes should be replaced by jobs.
Also the definition of the Abstract Test looks to state the same:
...
Issue an HTTP GET request to the URL ‘/jobs/{jobID}/results’ using an invalid {jobID}.
....

--- a/src/main/java/org/opengis/cite/ogcapiprocesses10/jobs/Jobs.java
+++ b/src/main/java/org/opengis/cite/ogcapiprocesses10/jobs/Jobs.java
@@ -171,7 +171,7 @@ public class Jobs extends CommonFixture {
 		    getJobsListURL = new URL(jobsListEndpointString);
 		    getProcessesListURL = new URL(processListEndpointString);
 		    getInvalidJobURL = new URL(processListEndpointString + "/invalid-job-" + UUID.randomUUID());
-		    getInvalidJobResultURL = new URL(rootUri +  getProcessListPath + "/invalid-job-" + UUID.randomUUID() + "/results");
+		    getInvalidJobResultURL = new URL(jobsListEndpointString + "/invalid-job-" + UUID.randomUUID() + "/results");
 			clientBuilder = HttpClientBuilder.create();
 		} catch (Exception e) {		
 			Assert.fail("Could set up endpoint: " + processListEndpointString + ". Exception: " + e);

